test/unit/config/test__base.py

```diff
@@ -314,3 +314,28 @@ def test_containerization_settings(tmpdir, container_runtime):
             assert '--user=' in rc.command[index]
         else:
             assert rc.command[index] == element
+
+
+@pytest.mark.parametrize(
+    'container_runtime, expected', (
+        ('docker', None),
+        ('podman', '1')
```

Comment:
We really need to just fix it so that these tests are actually run rather than doing this awkward mock.

Reply:
Or refactor the code so we can pass in a parameter for testing use rather than having to do all this setup.